reflex/.templates/jinja/custom_components/src.py.jinja2

```diff
@@ -42,12 +42,12 @@ class {{ component_class_name }}(rx.Component):
     # In this case you can use the lib_dependencies property to specify other libraries to install.
     # lib_dependencies: list[str] = []
 
-    # Event triggers, I did not understand the wording of the doc.
-    # def get_event_triggers(self) -> dict[str, Any]:
-    #     return {
-    #         **super().get_event_triggers(),
-    #         "on_change": lambda e0: [e0],
-    #     }
+    # Event triggers declaration if any.
+    # Below is equivalent to merging `{ "on_change": lambda e0: [e0] }`
+    # onto the default event triggers of parent/base Component.
+    # The function defined for the `on_change` trigger maps args for the javascript
+    # trigger to args that will be passed to the backend event handler function.
+    # on_change: rx.EventHandler[lambda e0: [e0]]
```

Comment:
small nit, but should we call this variable `e` rather than `e0`? Or maybe even something like `arg`, I'm thinking the original name may be confusing.